client/src/components/ui/Footer.js

```diff
@@ -32,7 +32,7 @@ class Footer extends Component {
                                 <div className="footer__column">
                                     <div className="footer__copyright">
                                         <Trans>copyright</Trans> &copy; {this.getYear()}{' '}
-                                        <a href="https://adguard.com/">AdGuard</a>
+                                        <a target="_blank" href="https://adguard.com/">AdGuard</a>
```

Comment:
According to ESlint rules we need to add `rel="noopener noreferrer"` to every link with `target="_blank"`

Reply:
I didn't know that, let me update.